### PR TITLE
Update Node 4.2 to 4.4 and small addition to vjsgenclean script

### PR DESCRIPTION
**Pull request by misteroneill**

### CODE REVIEW

File: bin/videojs-generator-cleanup.js [hunk 1]
```diff
@@ -25,6 +25,7 @@ var configs = {
       'devDependencies.portscanner',
       'devDependencies.serve-static',
       'devDependencies.watchify',
+      'scripts.build:test:browserify',
```
Comment (incompl) [inline bin/videojs-generator-cleanup.js]: Is this related to node 4.4?

Comment (misteroneill) [inline bin/videojs-generator-cleanup.js]: Nope. That's why the commit says "Sneak in..." 😄

Comment (incompl) [inline bin/videojs-generator-cleanup.js]: Gotcha. Can always change the PR title after the fact, too.

**Comment by incompl** [comment]
LGTM

**Pull request merged**